addons/editor/functions/fnc_deepCopy.sqf

```diff
@@ -0,0 +1,154 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Returns deep copy data for the given objects.
+ *
+ * Arguments:
+ * 0: Objects <ARRAY>
+ *
+ * Return Value:
+ * Object Data <ARRAY>
+ *
+ * Example:
+ * [_objects] call zen_editor_fnc_deepCopy
+ *
+ * Public: No
+ */
+
+params ["_objects"];
+
+// Filter destroyed objects and get the vehicle for mounted infantry
+_objects = _objects select {alive _x && {isNull isVehicleCargo _x}} apply {vehicle _x};
+_objects = _objects arrayIntersect _objects;
+
+// Get the world position where the cursor is pointing
+// Object positions are copied relative to the cursor
+private _centerPos = if (visibleMap) then {
+    private _ctrlMap = findDisplay IDD_RSCDISPLAYCURATOR displayCtrl IDC_RSCDISPLAYCURATOR_MAINMAP;
+    private _pos2D = _ctrlMap ctrlMapScreenToWorld getMousePosition;
+    _pos2D + [0]
+} else {
+    screenToWorld getMousePosition
+};
```

Comment:
Should be extracted into a new function.

Reply:
Was going to use `getPosFromScreen` after making the necessary changes to it.